stories/Components/Forms/Textpanel/FormTypes.jsx

```diff
@@ -0,0 +1,182 @@
+import React from "react";
+import "./FormTypes.scss";
+import { Inputcomponent } from "../Inputfields/Input_group";
+import { Image } from "../../../Atom/Images/Image/Image";
+import { Heading } from "../../../Atom/Typography/Heading/Heading";
+
+export const FormTypes = ({
+  element,
+  type,
+  required,
+  mode,
+  label,
+  button,
+  errorText,
+  placeholder,
+  placeholderemail,
+  anchorTag,
+  para,
+  ...args
+}) => (
+  <form>
+    <div className="grid-container form-types">
```

Comment:
@Ashish-srij Can we change class name from `form-types` to `form-wrapper`.

Reply:
done